contracts/deployables/erc20/VotesERC20LockableV1.sol

```diff
@@ -22,9 +24,7 @@ contract VotesERC20LockableV1 is IVotesERC20LockableV1, VotesERC20V1 {
         if (
             _locked &&
             // overrides while locked
-            !(from == owner() || // owner can always transfer
-                _whitelisted[from] || // whitelisted addresses can always transfer
-                from == address(0)) && // can always mint when locked
+            !hasRole(TRANSFER_ROLE, from) && // whitelisted addresses can always transfer
             to != address(0) // can always burn when locked
         ) {
             revert IsLocked();
```

Comment:
I'm a little confused on how this is working with the minter role. 

Since _mint calls _update, which we've overriden with this modifier. And in this modifier we aren't checking for the MINTER_ROLE. Maybe I'm missing something, but do you mind explaining that?

Reply:
The MINTER_ROLE is enforced in the mint function, as shown below:
```solidity
function mint(
    address to,
    uint256 amount
) external virtual override onlyRole(MINTER_ROLE) {
```
To elaborate further: both transfer and mint ultimately call _update, but transfer explicitly forbids either from or to from being address(0). Therefore, the only valid way to mint tokens is through the mint function, which is why the MINTER_ROLE check is implemented there.